pom.xml

```diff
@@ -62,7 +62,7 @@
         </pluginRepository>
     </pluginRepositories>
     <properties>
-        <jenkins.version>1.642.3</jenkins.version>
+        <jenkins.version>2.44-JENKINS-24141-SNAPSHOT</jenkins.version>
```

Comment:
I suspect you will need https://github.com/jenkinsci/plugin-pom/pull/59 (`deploy`ed as `2.27-20170501.155323-2`).